src/main/java/org/springframework/samples/petclinic/rest/VisitRestController.java

```diff
@@ -105,6 +106,8 @@
 		currentVisit.setDate(visit.getDate());
 		currentVisit.setDescription(visit.getDescription());
 		currentVisit.setPet(visit.getPet());
+        currentVisit.setAdHoc(visit.getAdHoc());
```

Comment:
Here we have the same thing about the business logic in the controllers. And also we can use an utility for copying the identical properties.